### DISCUSSION TITLE
[Feature] Search in albums

### DISCUSSION THREAD

**Discussion by Eidenz** [immich-app/immich#2407] [Feature Request]
### The feature

Would be great to have the search bar for albums as well. I share huge albums with some friends/family and I'd like them to be able to search within them.

### Platform

- [ ] Server
- [X] Web
- [X] Mobile

**Comment by SitramSoft** [comment]
I would also like this feature

**Comment by JPurrier** [comment]
Here is a similar one regarding search in mobile.  https://github.com/immich-app/immich/discussions/11835
Having the ability to search for albums in the mobile interface like you can in the web.

**Comment by anaxmedia** [comment]
I desperately need this on the mobile app.. I have hundreds of Albums and scrolling is slow and painful. Any ETA on when his might be implemented? It already exists on the web version.

**Reply by bo0tzz** [reply]
There is no ETA. Nobody is working on this at the moment.

**Reply by anaxmedia** [reply]
@bo0tzz is there any way to sponsor specific features to be implemented/prioritized? I would pay for this to be able to use the app as part of my workflow.

**Reply by bo0tzz** [reply]
We don't take feature sponsorships like that for the Immich team, but if you want to hire someone to implement it then that's an option. When doing that, please check in with the team first to make sure things will land on something we're willing to merge.

**Reply by JPurrier** [reply]
The ability to search album names in the mobile app like you can in the web interface would be a big win. I hope you can add it to the features you are looking to implement at some point soon.

**Comment by bo0tzz** [comment]
This is now implemented
